projects/croniter/fuzz_iter.py

```diff
@@ -17,32 +17,32 @@
 import sys
 
 from datetime import datetime
-import croniter
 
+with atheris.instrument_imports():
```

Comment:
Any particular reason changing this? I personally prefer `instrument_all`.

Reply:
This was a misguided attempt by me to reduce the coverage surface area a bit. Croniter is a fairly small library and is pulling in a lot of code coverage from its dependencies `dateutil` and `six`.

![image](https://user-images.githubusercontent.com/5122866/224102357-6a976a37-43bb-4766-852f-ae88a6a9bb62.png)

Reply:
This was a misguided attempt by me to reduce the coverage surface area a bit. Croniter is a fairly small library and is pulling in a lot of code coverage from its dependencies `dateutil` and `six`.

![image](https://user-images.githubusercontent.com/5122866/224102357-6a976a37-43bb-4766-852f-ae88a6a9bb62.png)